src/flame/variations/grid.ts

```diff
@@ -0,0 +1,27 @@
+import { random } from "@/shaders/random"
```

Comment:
do we want to have a subfolder `parametric`?

Reply:
Added all into a subfolder